bodhi/client/bindings.py

```diff
@@ -651,11 +670,14 @@ def update_str(self, update, minimal=False):
             return update
         if minimal:
             val = ""
+            security = '*' if update['type'] == u'security' else ' '
```

Comment:
I'm not sure why, but when I try to run the Bodhi client with this patch, the ' ' does not always make it into the final output in the CLI. It seems the formatting code might ```lstrip()``` it or something similar? For example, here's a query I ran on alembic:

```
[vagrant@bodhi-dev bodhi]$ bodhi updates query --packages alembic
*alembic-1.7.7-1.fc28                     rpm        testing   2018-04-01 (17)
 blender-2.79b-2.fc28                                                                                                                                                                                                                         
 openvdb-5.0.0-2.fc28                                                         
 YafaRay-3.3.0-9.fc28                                                                                                                                                             
*alembic-1.7.7-1.fc27                     rpm        testing   2018-04-01 (17)
 blender-2.79b-2.fc27                                                                                                                   
 openvdb-5.0.0-2.fc27                                                         
 YafaRay-3.3.0-9.fc27                                                                                                                                             
alembic-1.7.3-1.fc27                     rpm        stable    2017-09-30 (200)                                                                                                  
 blender-2.79-1.fc27                                                                                                   
 LuxRender-1.6-22.fc27                                                                                                                                   
 openvdb-4.0.2-1.fc27                                                          
 YafaRay-3.3.0-3.fc27                                                                                                                                      
alembic-1.7.3-1.fc26                     rpm        stable    2017-09-23 (207)                                                                                                  
 blender-2.79-1.fc26                                                                                                                                      
 LuxRender-1.6-22.fc26                                                                                                                                   
 openvdb-4.0.2-1.fc26                                                                                                                                                             
 YafaRay-3.3.0-3.fc26                                                                                                                                      
alembic-1.7.3-1.el7                      rpm        stable    2017-10-05 (194)                                                                           
alembic-1.7.3-1.fc25                     rpm        stable    2017-09-23 (206)
 blender-2.79-1.fc25                                                                                                                                              
 LuxRender-1.6-22.fc25
 openvdb-4.0.2-1.fc25
 YafaRay-3.3.0-3.fc25
alembic-1.7.1-3.fc24                     rpm        stable    2017-05-22 (331)
 blender-2.78c-4.fc24
 openvdb-4.0.1-2.fc24
alembic-1.7.1-3.fc25                     rpm        stable    2017-05-23 (329)
 blender-2.78c-4.fc25
 openvdb-4.0.1-2.fc25
alembic-1.7.1-3.el7                      rpm        stable    2017-05-26 (327)
alembic-1.7.1-3.fc26                     rpm        stable    2017-06-09 (313)
 blender-2.78c-4.fc26
 openvdb-4.0.1-2.fc26
10 updates found (10 shown)
```

Note that the non-security updates above were not preceeded by a space, but the security ones were preceeded by a ```*```.

Reply:
It would probably also be good for us to indent the list of other packages by one more space too (like ```blender``` in the above output).

Reply:
I woudn't be opposed to slapping the * after the update title if you don't want to figure out the spacing issue.

Reply:
> It seems the formatting code might lstrip() it or something similar?

Yes, in client/__init__.py on line 729:

    if 'updates' in resp:
        if len(resp.updates) == 1:
            click.echo(client.update_str(resp.updates[0]))
        else:
            for update in resp.updates:
                click.echo(client.update_str(update, minimal=True).strip())

I saw that, but I don't figure out why it's there... 
If it can be removed, I think it would not be so hard to add an extra space to other packages in the update.